web/src/features/time/HistoricalTimeHeader.tsx

```diff
@@ -23,7 +23,7 @@ import {
   timeAverageAtom,
 } from 'utils/state/atoms';
 
-const TIME_OFFSETS: Partial<Record<TimeAverages, number>> = {
+const WINDOW_OFFSETS: Partial<Record<TimeAverages, number>> = {
   [TimeAverages.HOURLY]: 24,
   [TimeAverages.HOURLY_72]: 72,
 };
```

Comment:
Is this still needed?

Reply:
We're still using it to calculate if the earliest datetime in a given historical window is within our historical limit